src/azul/openapi/__init__.py

```diff
@@ -14,98 +13,6 @@
 )
 
 
-def openapi_spec(path: str, methods: List[str], path_spec: Optional[JSON] = None, method_spec: Optional[JSON] = None):
-    """
-    Add description to decorated function at specified route.
-
-    >>> @openapi_spec('/foo', ['GET'], path_spec={'a': 'b'}, method_spec={'c': 'd'})
```

Comment:
Don't we want to retain these tests?

Reply:
I believe they are now redundant given that the `openapi_spec` decorator has been absorbed (in a dramatically simplified form) by `AuzlChaliceApp.route`, which is covered by `test_openapi.TestGetAppSpecs.` Do you have any specific tests for the route method that you'd like to see added?

Reply:
No. Just make sure that all cases currently covered by the doctest are covered by the test you mention.

Reply:
It looks like there was one test I missed. It's been added.